doc/nrf/releases/release-notes-latest.rst

```diff
@@ -75,6 +75,18 @@ nRF9160
     * Enabled all SLM services by default.
     * Updated the HTTP client service code to handle chunked HTTP responses.
 
+  * :ref:`aws_fota_sample` sample:
+
+    * Removed nRF Connect for Cloud support code, because ``fota_v1`` is no longer supported in nRF Connect for Cloud.
+    * Removed provisioning using :c:func:`modem_key_mgmt` and :file:`certificates.h`, because this is not the recommended way of provisioning private certificates.
```

Comment:
modem_key_mgmt doesn't seem to be a function. Do you mean the file? Or a specific function from that file?

Reply:
I was trying to refer to modem_key_mgmt_write and modem_key_mgmt_read. I think I forgot to add either. Maybe it's better to refer to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/include/modem/modem_key_mgmt.html ?

Reply:
Sure, might be the easiest.

Reply:
> I see a reference to nrf_cloud_app_fw_version in the image?

I have to find the code that generated the image or redraw it. Anyways it needs to be updated.

Reply:
There is a aws_fota_dfu_sequence.uml.txt file in the nrf/doc/nrf/images folder - that's probably the source.